spec/features/genres/visitor_can_see_genres_but_not_form_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+describe "As an admin" do
+  context "they visit the genre index" do
+    it "and fill in a form to create a new genre" do
+      genre1 = Genre.create!(name: 'Action')
+      genre2 = Genre.create!(name: 'Horror')
+      genre3 = Genre.create!(name: 'Romance')
+
+      visit genres_path
+
+      expect(current_path).to eq(genres_path)
+
+      assert has_no_field?(:name)
+
+      expect(page).to have_content(genre1.name)
+      expect(page).to have_content(genre2.name)
+      expect(page).to have_content(genre3.name)
+    end
+  end
+
+  context "they visit the genre index" do
+    it "and fill in a form to create a new genre" do
```

Comment:
This it block says the user is going to fill in the form, but that isn't what this test is about.